save-backend/src/test/kotlin/org/cqfn/save/backend/controller/TestSuitesControllerTest.kt

```diff
@@ -36,39 +40,48 @@ class TestSuitesControllerTest {
     @Test
     fun testConnection() {
         val project = projectRepository.findById(1).get()
-        val testSuite = TestSuite(
+        val testSuite = TestSuiteDto(
             TestSuiteType.PROJECT,
             "test",
-            project,
-            testLocalDateTime
+            project
         )
 
         webClient.post()
             .uri("/saveTestSuite")
             .contentType(MediaType.APPLICATION_JSON)
-            .body(BodyInserters.fromValue(testSuite))
+            .body(BodyInserters.fromValue(listOf(testSuite)))
             .exchange()
-            .expectStatus()
-            .isOk
+            .toMono()
```

Comment:
In tests you don't need to work with mono, you can call `expectBody` on `ResponseSpec` itself